modules/installation-and-upgrade/pages/container-deployment/uyuni/proxy-container-installation-uyuni.adoc

```diff
@@ -135,6 +135,30 @@ The volumes are named:
 
 To override default volume settings, create the volumes prior to the first start of the pod using the  [command]``podman volume create`` command.
 
+By default, the [package]``uyuni-proxy-systemd-services`` package creates these three directories that can be used to mount external disks:
```

Comment:
`uyuni-proxy-systemd-services` it's going to be deprecated. Better say something `mgrpxy` for uyuni (instead for 4.3 `uyuni-proxy-systemd-services`  is fine)